apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx

```diff
@@ -31,6 +31,7 @@ const chatService = new AzureOpenAI(AzureOpenAIConfig);
 async function getAIResponse(messages) {
   const params = {
     messages,
+    model: AzureOpenAIConfig.deployment,
```

Comment:
don't we want to apply this in all frameworks to get rid of ts warnings everywhere?

Reply:
[b83ec79eabe4b3dfb5b357bd0ea9b4bfe39a0859]